### PR TITLE
Updated the skip conditions to use IRuntimeEnvironment

### PR DESCRIPTION
**Pull request by victorhurdugaci**
cc @Praburaj

### CODE REVIEW

File: src/Microsoft.AspNet.Testing/project.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
     "version": "1.0.0-*",
     "dependencies": {
+        "Microsoft.Framework.Runtime": "1.0.0-*",
```
Comment (Praburaj) [inline src/Microsoft.AspNet.Testing/project.json]: `.Interfaces`?

Comment (davidfowl) [inline src/Microsoft.AspNet.Testing/project.json]: Yes, never rely on Runtime.

**Comment by victorhurdugaci** [comment]
Updated

**Comment by Praburaj** [comment]
Looks good to me :shipit:

**Pull request merged**